requirements.txt

```diff
@@ -15,7 +15,13 @@ PyMySQL==0.6.2
 DBUtils==1.1
 pyproj==1.9.3
 prettytable==0.7.2
+
 # When updating nupic.bindings, also update any shared dependencies to keep
 # versions in sync.
 nupic.bindings==0.5.1
 numpy==1.11.2
+
+# for documentation
+sphinx==1.5.3
+sphinx-autobuild==0.6.0
```

Comment:
You can add an entry to `setup.py`, a la "capnp" and "viz" (https://github.com/numenta/nupic/blob/master/setup.py#L164-L165).  That would allow someone the option of installing the sphinx/doc dependencies w/ setup.py or pip.